core/src/modules/ops/export.ts

```diff
@@ -106,6 +106,8 @@ export namespace ExportOps {
 
     _exports = await Export.findAll({
       where: {
+        completedAt: null,
+        errorMessage: null,
```

Comment:
@andyjih  this is a fix we jsut did that would prevent old, failed exports from re-processing.